modules/user_accounts/php/edit_user.class.inc

```diff
@@ -868,13 +868,50 @@ class Edit_User extends \NDB_Form
                     'N' => 'No'
                 )
             );
+            //------------------------------------------------------------
+
+            // active time windows
+
+            $dateOptions = array(
+                'language'       => 'en',
+                'format'         => 'YMd',
+                'addEmptyOption' => 'true',
+            );
+
+            $dateAttributes = ['class' => 'form-control input-sm input-date'];
+
+            $this->addBasicDate(
+                'active_from',
+                'Active from',
+                $dateOptions,
+                $dateAttributes
+            );
 
             $dateOptions = array(
                 'language'       => 'en',
                 'format'         => 'YMd',
                 'addEmptyOption' => 'true',
             );
 
+            // active
+            $this->addSelect('Active', 'Active', array('Y' => 'Yes', 'N' => 'No'));
+            $this->addSelect(
+                'Pending_approval',
+                'Pending approval',
+                array(
+                    'Y' => 'Yes',
+                    'N' => 'No',
+                )
+            );
+
+            // active time windows
+            $dateFormat  = $config->getSetting('dateDisplayFormat');
+            $dateOptions = array(
+                'language'       => 'en',
+                'format'         => $dateFormat,
+                'addEmptyOption' => 'true',
+            );
+
```

Comment:
something doesn't seem right with all of these changes. @johnsaigle could you verify and/or explain if this is intentional?

Reply:
I think I made a mistake while rebasing. I'll fix it.